Solver_interface/examples/Solver_interface/CMakeLists.txt

```diff
@@ -44,13 +44,13 @@ if ( CGAL_FOUND )
 
         target_link_libraries( mixed_integer_program PRIVATE ${GLPK_LIBRARIES} )
 
-        add_compile_definitions(CGAL_USE_GLPK)
+        add_definitions(-DCGAL_USE_GLPK)
```

Comment:
Be careful that `add_definitions` only applies to target defined **after** in the `CMakeLists.txt`. It look like it should be:
```cmake
target_compile_definitions(mixed_integer_program PRIVATE CGAL_USE_GLPK)
```